packages/auth/src/model/public_types.ts

```diff
@@ -199,6 +199,7 @@ export interface Auth {
    * @param persistence - The {@link Persistence} to use.
    */
   setPersistence(persistence: Persistence): Promise<void>;
+  setRecaptchaConfig(config: RecaptchaConfig): void;
```

Comment:
Please either add a jsdoc comment with a small snippet, or add a `TODO` with a bug number here to add appropriate documentation when it's available from the techwriters.